lib/internal/bootstrap/loaders.js

```diff
@@ -179,21 +179,33 @@ NativeModule._cache = {};
 
 const config = internalBinding('config');
 
+// Do not expose this to user land even with --expose-internals.
+const loaderId = 'internal/bootstrap/loaders';
+
+// Create a native module map to tell if a module is internal or not.
```

Comment:
Heads up: I think this does something similar but conflicts with https://github.com/nodejs/node/pull/25352?